### PR TITLE
Added Community Documents

### PR DESCRIPTION
**Pull request by zbarnz**
Added community documents both for our own reference and for the reference of outside contributors

## Sources
- Issue remplate - https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository
- PR template - https://github.com/axolo-co/pull_request_template
- Code of conduct - https://www.contributor-covenant.org/
- Contributing - written by me

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+:partying_face::cowboy_hat_face::tada::+1: Welcome to MeetingMinder :+1::tada::cowboy_hat_face::partying_face:
+
+## Resources
+- [Getting Started / Setup](https://github.com/SocexSolutions/agenda-v2/blob/master/README.md)
+- [Docs](https://github.com/SocexSolutions/agenda-v2/tree/master/docs)
+- [Issues](https://github.com/SocexSolutions/agenda-v2/issues)
+
+## Summary
+
+When contributing to MeetingMinder, if there is not already an issue open, please first discuss the change you wish to make via issue before making a change.
```
Comment (thudsonbu) [inline CONTRIBUTING.md:12]:
```suggestion
When contributing to MeetingMinder, if there is not already an issue open, please first discuss the change you wish to make via an issue before making a change. 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+### Frontend:
+
+- Spend some time testing what your changes affected. 
+- Try to break your own feature. 
+- Think about edge cases.
+
+### Backend:
+- make sure any additional features or changes to existing features are covered under the tests in `api/test/tests`
+- run `npm run test` in console and confirm all tests are passing
+- ensure changes function as expected on the front end
```
Comment (thudsonbu) [inline CONTRIBUTING.md:27]: Should these all be capitalized? Also, I think there should be a newline after `### Backend` for consistency.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+## Pull Request Process
+
+1. Write your pull request description according to the [pull request template](https://github.com/SocexSolutions/agenda-v2/blob/master/pull_request_template.md)
+2. Assign your pull request to a SocexSolutions developer under the github assign tab. This developer will handle assigning reviewers
+3. Reply to discussions. There may be cases where a developer requests a change before the Pull Request can be merged.
+4. You may merge the Pull Request in once you have the sign-off of one other SocexSolutions developer, or if you do not have permission to do that, you may request the reviewer to merge it for you.
+
+## Bug Reports
+
+### Before reporting
```
Comment (thudsonbu) [inline CONTRIBUTING.md:38]:
```suggestion
### Before Reporting
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- See if you can find the cause of the problem. You might be able to fix the problem and score an easy contribution
+- Check the [issues](https://github.com/SocexSolutions/agenda-v2/issues) page for similar existing bug reports
+
+### Submitting a report:
```
Comment (thudsonbu) [inline CONTRIBUTING.md:42]:
```suggestion
### Submitting a Report:
```

**Review by thudsonbu — APPROVED** [review]
